tox.ini

```diff
@@ -73,7 +74,6 @@ commands =
 
 [testenv:rust]
 basepython = python3
-skip_install = True
```

Comment:
Do we need to install here now because `cargo test` exercises things that otherwise can't be built?

Reply:
correct -- building the rust package now involves building our cffi code